apm-agent-plugins/apm-spring-webflux/apm-spring-webflux-spring5/src/test/java/co/elastic/apm/agent/springwebflux/AbstractServerInstrumentationTest.java

```diff
@@ -202,7 +202,7 @@ private Predicate<Throwable> expectClientError(int expectedStatus) {
     }
 
     @ParameterizedTest
-    @CsvSource({"GET", "POST", "PUT", "DELETE", "PATCH", "HEAD", "OPTIONS", "TRACE"})
+    @CsvSource({"GET", "POST", "PUT", "DELETE", "PATCH", "HEAD", "OPTIONS"})
```

Comment:
Looks like in spring 3.4.0 the `TRACE` method is now rejected by default with `400 BAD_REQUEST`. TBH I've so far never seen the `TRACE` method used in the wild, so imo it is okay to simply remove it from the tests.